internal/pkg/skuba/deployments/ssh/kernel.go

```diff
@@ -35,6 +35,10 @@ var (
 			Attribute: "net.ipv4.ip_forward",
 			Value:     "1",
 		},
+		"net-ipv4-conf-all-forwarding": {
+			Attribute: "net.ipv4.conf.all.forwarding",
```

Comment:
What is the difference with `net.ipv4.ip_forward` that we were already setting? Is this providing something extra? Should we remove `net.ipv4.ip_forward` and leave only this one?

Reply:
Maybe @mrostecki can pitch in here, but according to his comments in the issue, we need to check for both of those.

Reply:
`net.ipv4.ip_forward` enables IP forwarding, `net.ipv4.conf.all.forwarding` makes sure that all network devices have it enabled. Without the second sysctl, there might be a situation when i.e. `net.ipv4.conf.eth0.forwarding` has value `0` and then Cilium will not work.